selfdrive/car/hyundai/values.py

```diff
@@ -648,6 +651,8 @@ class Buttons:
       b'\xf1\x87KMMYBU034207SB72x\x89\x88\x98h\x88\x98\x89\x87fhvvfWf33_\xff\x87\xff\x8f\xfa\x81\xe5\xf1\x89HT6TAF00A1\xf1\x82STM0M25GS1\x00\x00\x00\x00\x00\x00',
       b'\xf1\x87954A02N250\x00\x00\x00\x00\x00\xf1\x81T02730A1  \xf1\x00T02601BL  T02730A1  VTMPT25XXX730NS2\xa6',
       b'\xf1\x00HT6TA290BLHT6TAF00A1STM0M25GS1\x00\x00\x00\x00\x00\x006\xd8\x97\x15',
+      b'\xf1\x00T02601BL  T02900A1  VTMPT25XXX900NS8\xb7\xaa\xfe\xfc',
+      b'\xf1\x87954A02N250\x00\x00\x00\x00\x00\xf1\x81T02900A1  \xf1\x00T02601BL  T02900A1  VTMPT25XXX900NS8\xb7 \xaa\xfe\xfc',
```

Comment:
This was either a typo, or you copied from a route where this FW version wasn't set correctly in time when we read it from the ECU. Let's only add the reliable FW version above for transmission.

For future reference: shorter response comes from request `[b'"\xf1\x00']`, longer response comes from: `[b'"\xf1\x87\xf1\x81\xf1\x00']`

Reply:
Actually I don't see your second FW version at all, but there is a longer FW version we're returning sometimes that's always the same so I added that.